docs/data/destinations/braze.md

```diff
@@ -45,10 +45,10 @@ After you create the destination, you must configure the settings.
 2. Select your **Braze REST API Endpoint**.
 3. Enter your **REST API Key**.
 4. Enter your **Braze App Identifier API Key**.
-5. **Create & Update Users** creates users in Braze and update the properties of existing users when an Amplitude Identify API call is made.
+5. **Create & Update Users** creates users in Braze and update the properties of existing users when a user is created or user properties are updated in Amplitude.
```

Comment:
Sanity check. Is sentence this confusing?

Reply:
A little. How does an existing user get created? Does this mean creates or (inclusive `or`) updates users in Braze when their corresponding user is created or updated in Amplitude?

```suggestion
5. Configure **Create & Update Users**. This setting does two things: creates users in Braze when they're created in Amplitude and updates existing users when their user properties are updated in Amplitude.
```

Reply:
@sina-jamshidi I made a suggestion that hopefully makes this more clear in the docs.